package.json

```diff
@@ -80,6 +80,7 @@
     "solidity-coverage": "^0.8.2"
   },
   "dependencies": {
+    "@biconomy-devx/paymasters": "^1.0.0",
```

Comment:
https://github.com/bcnmy/biconomy-paymasters/pull/37#discussion_r1355941325

Reply:
removed this dependency now to publish a package